app/models/episode.rb

```diff
@@ -210,4 +210,14 @@ def media_files
   def audio_files
     media_files
   end
+
+  def set_adzerk_keyword
+    return unless published? && adzerk_keyword.nil?
```

Comment:
sure, that sounds good. I'm guessing it's unlikely we'll have episodes that have a set `published_at` but no `title` yet.